test/isIndexed.js

```diff
@@ -0,0 +1,28 @@
+import { assert } from 'chai';
+
+import * as RA from '../src';
+
+describe('isIndexed', function() {
+  context('given a `String` value', function() {
+    specify('should return true', function() {
+      assert.isTrue(RA.isIndexed('test'));
+      assert.isTrue(RA.isIndexed(Object('test')));
+    });
+  });
+
+  context('given array value', function() {
+    specify('should return true', function() {
+      assert.isTrue(RA.isIndexed([1, 2]));
+    });
+  });
+
+  context('given a non `String` or `Array` value', function() {
+    specify('should return false', function() {
+      assert.isFalse(RA.isIndexed(true));
+      assert.isFalse(RA.isIndexed(new Date()));
+      assert.isFalse(RA.isIndexed(new Error()));
+      assert.isFalse(RA.isIndexed({ 0: 1, length: 1 }));
+      assert.isFalse(RA.isIndexed(1));
+    });
+  });
+});
```

Comment:
We're missing test for currying in form on ramda placeholder. See how it should be done here: https://github.com/char0n/ramda-adjunct/blob/a75aa870faf3356e73c7de5fc3e7d5dbae1e071b/test/isFalse.js#L43

Reply:
Something like this?
```
 it('should support placeholder to specify "gaps"', function() {
    const isIndexed = RA.isIndexed(R.__);

    assert.isTrue(isIndexed([1, 2]));
  });
```
This doesn't work for me so i'll need to research this more

Reply:
Take a look at this for example: https://github.com/char0n/ramda-adjunct/blob/86b7e1ca88b5f170ad977d930dc907a61734ad68/src/isNegative.js#L26

Your `inIndexed` at this moment is not curried. We have to wrap it into `curry1`. More info about ramda currying here: https://fr.umio.us/favoring-curry/

Reply:
ah thanks!

Reply:
I, my other PR i curried it, using `curry`, not `curry1`. I'm not sure  if there's any difference? e.g. `paths.js` uses `curry`

Reply:
Excelusively for unary functions (functions with one argument) we historically used ramdas internal curry1 function which is faster then public `curry` or `curryN` functions. But it created some additional problems.

Check out this [issue](https://github.com/char0n/ramda-adjunct/issues/1340) for further explanation.